### DISCUSSION TITLE
Resolution Huge, Game Tiny Question.

### DISCUSSION THREAD

**Discussion by DHBgithub2021** [diasurgical/DevilutionX#5319] [General]
First, fantastic job on this. Thank you. If this question has been addressed elsewhere I apologize, I am not really sure how to ask.

So, I have a large-ish monitor. 3840 x 2160 is my display resolution. When I play the game at that resolution everything is very distant and tiny, even when zoomed. So basically I am playing from the perspective of an aerial reconnaissance video. 

I am probably not doing something in the settings to correct this. If someone could explain how to have a high resolution with the "normal", original game screen coverage I would be very grateful.

Thank you to anyone offering assistance, I appreciate the help.

**Comment by Trihedraf** [comment]
Set the resolution to 640x480 and enable upscale and it will have the right zoom level.

**Reply by wisac** [reply]
Try playing at a lower resolution. Maybe would work. Try 1920 x 1080  or lower.

**Comment by AJenbo** [comment]
> I am probably not doing something in the settings to correct this. If someone could explain how to have a high resolution with the "normal", original game screen coverage I would be very grateful.

That is the default settings. Resolution is the internal game area, not your desired output resolution.
